app/models/miq_queue.rb

```diff
@@ -101,10 +101,7 @@ def self.put(options)
     options = options.reverse_merge(
       :priority     => NORMAL_PRIORITY,
       :queue_name   => "generic",
-      :role         => nil,
-      :server_guid  => nil,
       :msg_timeout  => TIMEOUT,
-      :deliver_on   => nil
```

Comment:
Seems like this is the only _**actual**_ code change to the application code, and effectively it was just removing something that was `nil` regardless, right?